src/link.js

```diff
@@ -0,0 +1,25 @@
+export function baseLink(element, parser, options) {
+  const {Link} = parser.minim.elements;
+  options = options || {}
+
+  const link = new Link();
+  link.relation = options.relation || 'origin';
+  link.href = `http://docs.apiary.io/validations/swagger#${(options.fragment || '')}`;
```

Comment:
These links are 404, should there be anything here? Shouldn't this be a link to a specification about this element type?

They are also against docs.apiary.io which was moved to help.apiary.io